plugins/Live/tests/Fixtures/VisitsWithAllActionsAndDevices.php

```diff
@@ -32,13 +33,20 @@ public function setUp(): void
         CustomDimensionsApi::getInstance()->configureNewCustomDimension(1, 'currency', 'action', 1);
 
         $t = self::getTracker($this->idSite, $this->dateTime, $defaultInit = true);
+
+        // Campaign - needs a separate user otherwise it overrides the referrer
         $t->setTokenAuth(self::getTokenAuth());
+        $t->setUserId('Z4F66G776HGI');
+        $t->setVisitorId(substr(sha1('Z4F66G776HGI'), 0, 16));
```

Comment:
If you want to create a new visitor the simplest way is to recreate the tracker instance with
`$t = self::getTracker($this->idSite, $this->dateTime, $defaultInit = true);`
That way all other possibly set parameters will also be unset.

Reply:
Thanks, I'll make a note of that.